drivers/gpio/Kconfig

```
@@ -208,6 +208,6 @@ source "drivers/gpio/Kconfig.sedi"
 
 source "drivers/gpio/Kconfig.altera"
 
-source "drivers/gpio/Kconfig.bcm2711"
+source "drivers/gpio/Kconfig.kb1200"
```

Should this be:

```suggestion
source "drivers/gpio/Kconfig.bcm2711"
source "drivers/gpio/Kconfig.kb1200"
```
?